source/atlas-cli-deploy-fts.txt

```diff
@@ -0,0 +1,529 @@
+                  :language: sh
+
+                  [
+                    {
+                      plot: 'A trio of guys try and make up for missed opportunities in childhood by forming a three-player baseball team to compete against standard children baseball squads.',
+                      title: 'The Benchwarmers'
+                    },
+                    {
+                      plot: 'A young boy is bequeathed the ownership of a professional baseball team.',
+                      title: 'Little Big League'
+                    },
+                    {
+                      plot: 'A trained chimpanzee plays third base for a minor-league baseball team.',
+                      title: 'Ed'
+                    },
+                    {
+                      plot: 'The story of the life and career of the famed baseball player, Lou Gehrig.',
+                      title: 'The Pride of the Yankees'
+                    },
+                    {
+                      plot: 'Babe Ruth becomes a baseball legend but is unheroic to those who know him.',
+                      title: 'The Babe'
+                    }
+                  ]
+
+.. _atlas-cli-deploy-avs:
+
+Use {+avs+} with a Atlas Deployment
+-----------------------------------------------
+
+Use the ``atlas deployments`` command to work with {+avs+}. To 
+learn more, see :atlas:`How to Index Vector Embeddings for Vector 
+Search </atlas-search/field-types/knn-vector>`.
+
+.. include:: /includes/fact-avs-mdb-version.rst
+
+.. procedure::
+   :style: normal
+
+   .. step:: Create an ``indexDef-vector.json`` configuration file.
+
+      .. code-block:: json
+
+         {
+           "collectionName": "embedded_movies",
+           "database": "sample_mflix",  
+           "mappings": {
+             "dynamic": true,
+             "fields": {
+               "plot_embedding": {
+                 "type": "knnVector",
+                 "dimensions": 1536,
+                 "similarity": "euclidean"
+               }
+             }
+           },
+           "name": "vectorSearchIndex"
+         }
+   
+   .. step:: Create an |fts| index.
+
+      a. Run the following command:
+
+         .. code-block:: sh
+
+            atlas deployments search indexes create --file indexDef-vector.json
+
+      #. Specify the deployment and press ``Enter``.
+
+   .. step:: Connect to the deployment.
+
+      a. Run the following command to connect to a deployment:
+
+         .. code-block:: sh
+
+            atlas deployments connect
+      
+      #. Specify the deployment to connect to and press 
+         ``Enter``.
+
+      #. Specify ``mongosh`` and press ``Enter``.
+
+   .. step:: Switch to the ``sample_mflix`` database.
+
+      .. code-block:: sh
+
+         use sample_mflix
+
+   .. step:: Run an example query.
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+            :language: sh
+            
+            db.embedded_movies.aggregate([
```

Comment:
how are we wording this tutorial in other docs? I am not able to copy this command into the shell and I remember this was an issue that mongosh wanted to fix. as of now mongosh gets stuck for me

Reply:
@blva, it does take a while to copy and paste such a long vector query into the command line. It should paste over eventually. @kanchana-mongodb, does the Search team have any thoughts on this?

Reply:
In my testing also, it takes a while to copy/paste, but has never gotten stuck. I can ask the search team about this, but doubt there's much they can do about the delay in being able to paste the query into the terminal.

Reply:
cool! great to know. should we add a warning/comment for customers to ask them to wait? would be nice to have a time estimate as well

Reply:
@blva, sure I'll add an important note that copying and pasting a large query vector might take a while depending on your machine.